src/components/VmDetails/cards/NicsCard/NicListItem.js

```diff
@@ -44,7 +44,7 @@ const NicListItem = ({ idPrefix, nic, vmStatus, vnicProfileList, isEditing, onEd
       <Grid>
         <Row>
           <Col cols={4} wrapExpand className={style['ip4-container']} id={`${idPrefix}-ipv4`}>
-            { nic.ipv4.length > 0 &&
+            { vmStatus === 'up' && nic.ipv4.length > 0 &&
```

Comment:
This is already handled here: https://github.com/oVirt/ovirt-web-ui/blob/4e018d11bb74d118315804b36949c501e04b7ef5/src/components/VmDetails/cards/NicsCard/index.js#L101